src/components/Dashboard.vue

```diff
@@ -157,6 +164,11 @@ export default {
       coverStyle: {}
     }
   },
+  watch: {
+    pickedTopic: function () {
+      this.pickedSubtopic = ''
```

Comment:
Could you add a brief explanation for this watcher to describe that it resets the subtopics when a new topic is chosen?